### PR TITLE
publish r31 from next into main

### PR DESCRIPTION
**Pull request by CaroMac**

### CODE REVIEW

File: src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ On Mac or Unix:
 ```
 galasactl runs submit local --log - \
 --obr mvn:dev.galasa/dev.galasa.simbank.obr/0.25.0/obr \
---obr mvn:dev.galasa/dev.galasa.uber.obr/0.30.0/obr \
+--obr mvn:dev.galasa/dev.galasa.uber.obr/0.31.0/obr \
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md:113]: pretty sure you don't need that line.

File: src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ On Windows (Powershell):
 ```
 galasactl runs submit local --log - `
 --obr mvn:dev.galasa/dev.galasa.simbank.obr/0.25.0/obr `
---obr mvn:dev.galasa/dev.galasa.uber.obr/0.30.0/obr `
+--obr mvn:dev.galasa/dev.galasa.uber.obr/0.31.0/obr `
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md:122]: pretty sure you don't need to add this line.

File: src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md [hunk 3]
```diff
@@ -85,14 +85,14 @@ On Mac or Unix:
...
---obr mvn:dev.galasa/dev.galasa.uber.obr/0.30.0/obr \
+--obr mvn:dev.galasa/dev.galasa.uber.obr/0.31.0/obr \
 --class dev.galasa.simbank.tests/dev.galasa.simbank.tests.SimBankIVT 
 ```
 On Windows (Powershell):
 ```
 galasactl runs submit local --log - `
 --obr mvn:dev.galasa/dev.galasa.simbank.obr/0.25.0/obr `
---obr mvn:dev.galasa/dev.galasa.uber.obr/0.30.0/obr `
+--obr mvn:dev.galasa/dev.galasa.uber.obr/0.31.0/obr `
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md:95]: pretty sure you don't need that line.

File: src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md [hunk 4]
```diff
@@ -85,14 +85,14 @@ On Mac or Unix:
 ```
 galasactl runs submit local --log - \
 --obr mvn:dev.galasa/dev.galasa.simbank.obr/0.25.0/obr \
---obr mvn:dev.galasa/dev.galasa.uber.obr/0.30.0/obr \
+--obr mvn:dev.galasa/dev.galasa.uber.obr/0.31.0/obr \
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md:88]: pretty sure you don't need that line.

File: src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md [hunk 5]
```diff
@@ -29,7 +30,54 @@ The CPS is a key-value pair store. Properties in the CPS are dot-separated value
...
+
+```
+galasactl properties namespaces get
+```
+
+The following example shows namespaces returned in summary format:
+
+```
+name       type
+docker     normal
```
Comment (techcobweb) [inline src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md:62]: The types are always upper case:
```
namespace       type
kubernetes      NORMAL
hbank           NORMAL
zosprogram      NORMAL
githubissue     NORMAL
mcobbett        NORMAL
secure          SECURE
docker          NORMAL
java            NORMAL
simbank         NORMAL
linux           NORMAL
zos3270         NORMAL
sem             NORMAL
rseapi          NORMAL
zosmf           NORMAL
jat             NORMAL
beta            NORMAL
zos             NORMAL
xyz123          NORMAL
test            NORMAL
core            NORMAL
framework       NORMAL
cicsts          NORMAL
galasaecosystem NORMAL
openstack       NORMAL
phoenix2        NORMAL
ecosystemtest   NORMAL
```

File: src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md [hunk 6]
```diff
@@ -29,7 +30,54 @@ The CPS is a key-value pair store. Properties in the CPS are dot-separated value
...
+Use the following command to retrieve all namespaces in an Ecosystem `raw` format:
+
+```
+galasactl properties namespaces get --format raw
+```
+
+The following example shows namespaces returned in the `raw` format of `namespace|type|`:
+
+```
+docker|normal|framework|normal|secure|secure|
```
Comment (techcobweb) [inline src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md:77]: Each namespace appears on a separate line:
```
>galasactl properties namespaces get --format raw

kubernetes|NORMAL
hbank|NORMAL
zosprogram|NORMAL
githubissue|NORMAL
mcobbett|NORMAL
secure|SECURE
docker|NORMAL
java|NORMAL
simbank|NORMAL
linux|NORMAL
zos3270|NORMAL
sem|NORMAL
rseapi|NORMAL
zosmf|NORMAL
jat|NORMAL
beta|NORMAL
zos|NORMAL
xyz123|NORMAL
test|NORMAL
core|NORMAL
framework|NORMAL
cicsts|NORMAL
galasaecosystem|NORMAL
openstack|NORMAL
phoenix2|NORMAL
ecosystemtest|NORMAL
```

File: src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md [hunk 7]
```diff
@@ -29,7 +30,54 @@ The CPS is a key-value pair store. Properties in the CPS are dot-separated value
...
+Total: 3
+```
+
+Use the following command to retrieve all namespaces in an Ecosystem `raw` format:
+
+```
+galasactl properties namespaces get --format raw
+```
+
+The following example shows namespaces returned in the `raw` format of `namespace|type|`:
```
Comment (techcobweb) [inline src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md:74]: ie: With 'pipe' delimeters

**Review by techcobweb — APPROVED** [review]
Approved, subject to those issues being sorted out.

File: src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md (hunk 1 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md:113]: thanks - removed

File: src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md (hunk 2 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md:122]: thanks - removed

File: src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md (hunk 3 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md:95]: thanks - removed

File: src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md (hunk 4 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md:88]: thanks - removed

File: src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md (hunk 5 above)
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md:62]: updated

File: src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md (hunk 6 above)
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md:77]: updated

File: src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md (hunk 7 above)
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md:74]: added

**Review by KirbyKatcher — APPROVED** [review]

**Pull request merged**